pkg/expr/func_arr.go

```diff
@@ -220,3 +222,59 @@ func slice(arr interface{}, start, end int) interface{} {
 
 	return v.Slice(start, end).Interface()
 }
+
+// sortSlice sorts slice
+func sortSlice(arr interface{}, desc bool) (out interface{}, err error) {
+	if arr, err = toSlice(arr); err != nil {
+		return
+	}
+
+	// sort slice of typed values
+	if stv, is := arr.([]TypedValue); is {
+		return sortSliceStable(stv, func(i, j int) bool {
+			c, _ := stv[i].(Comparable).Compare(stv[j])
```

Comment:
This expression scares me; a typed value does not necessarily implement the `Comparable` interface so this line will panic because you are not "respecting" that fact (for lack of a better word).

To avoid a panic you need to do something like this:

```go
casted, ok := thing.(Comparable)
if !ok {
    // Does not conform; don't try to compare it
}
```

(omitting the `ok` panics if it doesn't conform).